src/sentry/static/sentry/less/shared-components.less

```diff
@@ -1918,6 +1918,16 @@ ul.faces {
     }
   }
 
+  .invite-member a {
```

Comment:
so, on one hand we really don't want to add new things to shared-components if we can help it, because it's hard to delete from and it's global.

I think the best way to style this would be to use the `InlineSvg` component instead of the icon-plus classname, and then apply the remaining styles using react-emotion 

on the other hand, the plan is for this whole component to be re-done to use `DropdownAutoComplete`, so it might be okay for this to live here for now

Reply:
Could you elaborate on the shared-components bit? I'm not sure I follow the cons.

Ok will leave this here for now. InlineSvg does seem cleaner though.

Reply:
i think its probably ok for now

Reply:
agreed that it's fine, but the cons to shared-components are that they're global styles and so they are either very specifically tied to component nesting structure and make the connection between element hierarchy and style brittle, or they may be overly broad and accidently match things you don't intend. 

Because of these issues, it's hard to know when it's safe to change or remove things in this file and so it becomes an append-only mess :/